js-intro.js

```diff
@@ -4,39 +4,63 @@
+// -----COMPLETE-----1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
 // Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
-
-
-// --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
+// create a function that determines if a given temp is below boiling point (bp<212), at boiling point (=212), or above boiling point (bp>212)
+var boilingPoint = (temp) => {
+    if (temp<212) {
+        return (`${temp} is below boiling point`)
+    } else if (temp>212) {
+        return (`${temp} is above boiling point`)
+    } else{
+        return (`${temp} is at boiling point`)
+    }
+}
+
+console.log(boilingPoint(temp1));
+console.log(boilingPoint(temp2));
+console.log(boilingPoint(temp3));
+
+// ----COMPLETE------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
-var myNumbers1 = [3, 7, 0, 6, -9]
-
+//create function that multiplies each number in array by 5 using a for loop
 
+var myNumbers1 = [3, 7, 0, 6, -9]
 
+for(let i=0; i<myNumbers1.length; i++) {
+    console.log(myNumbers1[i] *5)
+}
 
-// --------------------3) Create a function that multiplies each number in the array by 5 using map.
+// -----COMPLETE-----3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
+//create a function that multiplies each number in the array by 5 using map
+var newArr = myNumbers2.map(function(value) {
+    return value * 5
+})
+console.log (newArr)
 
-
-
-
-// --------------------4) Create a function that removes all the vowels from a string.
+// -----COMPLETE------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
-var stringWithVowels1 = "HeyThereLearnStudent"
-var stringWithVowels2 = "ILoveJavaScript"
-
-
+var stringWithVowels1 = "HeyThereLearnStudent";
+var newarrayWithVowels1 = stringWithVowels1.split("");
+var stringWithVowels2 = "ILoveJavaScript";
+var newarrayWithVowels2 = stringWithVowels2.split("");
+// create a function that removes all vowels from a string
+const noVowels1 =newarrayWithVowels1.filter(x => x !=="a" && x!=="e" && x!=="i" && x!=="o" && x!=="u")
+const noVowels2 =newarrayWithVowels2.filter(x => x !=="a" && x!=="e" && x!=="i" && x!=="o" && x!=="u" && x!=="I")
+//Expected output: "HyThrLrnStdnt"
+console.log(noVowels1.join(""));
+//Expected output: "LvJvScrpt"
+console.log(noVowels2.join(""));
```

Comment:
This logic technically works with the given examples but again, it needs to be wrapped in a function and made to be reusable with _any_  string we decide to pass to it.